pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -582,8 +591,17 @@ func userInfoToExternalClaims(userInfo *userInfoType) *tokens.ExternalUserClaim
 		claim.UserID = userInfo.EMail
 	}
 
-	// Add all fields as attributes.
+	// Add sso.redhat.com attributes.
 	claim.Attributes = make(map[string][]string)
+	realmAccess := userInfo.RealmAccess
+	if realmAccess != nil && len(realmAccess.Roles) > 0 {
+		claim.Attributes[authproviders.RolesAttribute] = realmAccess.Roles
```

Comment:
What are we going to do with this new attribute? Extend our auth provider rule logic to also support roles? I'd suggest to scan roles for `admin:org:all` and add it to the groups attribute and ignore all the junk.